sdk/turing/router/config/route.py

```diff
@@ -37,7 +40,6 @@ def endpoint(self) -> str:
 
     @endpoint.setter
     def endpoint(self, endpoint: str):
-        Route._verify_endpoint(endpoint)
```

Comment:
validation is now done at `RouteConfig` init instead, since there is no more `Protocol` field for Routes.